src/python/WMCore/Services/WorkQueue/WorkQueue.py

```diff
@@ -61,7 +61,7 @@ def getJobStatusByRequest(self):
         data = self.db.loadView('WorkQueue', 'jobStatusByRequest',
                                 self.defaultOptions)
         return [{'request_name' : x['key'][0], 'status': x['key'][1],
-                 'jobs' : x['value']} for x in data.get('rows', [])]
+                 'jobs' : x['value']['sum']} for x in data.get('rows', [])]
```

Comment:
I was suspecting this function is used somewhere outside WMCore. I couldn't find any usage outside unittest. 
So I didn't want to change the return type. But I couldn't find any usage from WmAgentScript. So I guess we can updated that.